src/management/index.js

```diff
@@ -757,7 +757,7 @@ utils.wrapPropertyMethod(ManagementClient, 'deleteRule', 'rules.delete');
 
 
 /**
- * Delete an existing rule.
+ * Update an existing rule.
```

Comment:
can you also upgrade the example right below this line?

Reply:
sure, thanks for pointing out. I was investigating the updateEmailProvider and came across that line, haven't noticed that it needed some changes.